src/main/java/budgetbuddy/model/account/Account.java

```diff
@@ -48,16 +50,43 @@ public Transaction getTransaction(Index toGet) {
         return this.transactionList.getTransaction(toGet);
     }
 
+    public long getBalance() {
+        return balance;
+    }
+
+    /**
+     * Add a transaction to the transactionList.
+     * @param toAdd
+     */
     public void addTransaction(Transaction toAdd) {
         this.transactionList.add(toAdd);
+        if (toAdd.getDirection().toString().equals("IN")) {
```

Comment:
Since `Direction` is an enum, you can do `toAdd.getDirection() == Direction.IN`.

Reply:
Perhaps use `toAdd.getDirection().equals(Direction.IN)` instead